apps/api/src/server/applications/application/application.service.js

```diff
@@ -5,7 +5,6 @@ import { breakUpCompoundStatus } from '../../utils/break-up-compound-status.js';
 import { buildAppealCompundStatus } from '../../utils/build-appeal-compound-status.js';
 import { mapApplicationDetails } from '../../utils/mapping/map-case-details.js';
 import { mapFolderDetails } from '../../utils/mapping/map-folder-details.js';
-// import { sendMessage } from '../../utils/service-bus-sender.js';
```

Comment:
Could we leave this in please? This is a placeholder for when we will be sending messages to the message queue

Reply:
i didn't remove this